### ISSUE TITLE
[XamarinProfiler-MAC]: Accessibility: Voiceover: MAS4.2.1: The name property of the "Choose Target" button is not being announced by the VoiceOver.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#433]
### Steps to Reproduce

1. Install Xamarin Profiler.
2. Launch Visual Studio.
3. Create a project and build the solution.
4. Navigate to Run Menu Tab and Invoke it. Navigate to "Start Profiling" option and invoke it.
5. This will invokes "Xamarin Profiler" application."
6. Navigate to the "Choose Target" button of the Xamarin Profiler window.
### Expected Behavior
The VoiceOver should announce as "Choose Target" button.
### Actual Behavior
The VoiceOver  announces as  button, but Name of the button is not being announced by the VoiceOver.
### User Impact
The User won't be able to know the purpose of the button until its name is not being announced
### Screenshot
![voiceover_profilermac1](https://user-images.githubusercontent.com/22956130/41714337-a43fdc90-756d-11e8-858c-270c4d658acb.png)

### Environment
macOS High Sierra OSX version: 10.13.4
Visual Studio for Mac Enterprise version: 7.5.2(build 40)
Xamarin Profiler version: 1.6.3-400

**Comment by WiproPSAccTest** [comment]
Please add the following tags to this bug:
A11yMAS
Accessibility
MAS4.2.1
Xamarin-Profiler-MAC
Wipro
